xonsh/completer.py

```diff
@@ -201,24 +210,29 @@ def find_and_complete(self, line, idx, ctx=None):
             endidx += 1
         endidx = endidx - 1 if line[endidx] in WS else endidx
         prefix = line[begidx:endidx+1]
-        return self.complete(prefix, line, begidx, endidx, ctx=ctx), begidx, endidx
+        return (self.complete(prefix, line, begidx, endidx, ctx=ctx),
+                begidx, endidx)
 
     def _add_env(self, paths, prefix):
+        _ = self
```

Comment:
The alternative is to either make this method a module level function or wrap it in `@staticmethod`. The rule of thumb within Google is to avoid `@staticmethod` and use the idiom I've introduced here if the function is tightly bound to the class otherwise move it to be a method level function.